### ISSUE TITLE
Finish editing a cell on mouse click on a dummy row?

### ISSUE THREAD

**Issue by SanjaySankar** [nhn/tui.grid#1058]
As per the **editor** that the grid provides, the value edited by the user is **saved as the new value** of the cell by either: 

-  pressing the **Enter key**, or
- **clicking on another part of the table that has data**.

If we **click on the dummy rows** that gets populated in the table when there is empty space, **the edited value doesn't get saved**. 

**Is there a way to save the value on clicking on a dummy row as well?** Becase when I have only one row of data in the table, I have to either press enter or click on that one row to save the value.

**Comment by js87zz** [comment]
@SanjaySankar 
The editing value is saved by pressing enter key or changing the focused cell as you commented.
However, the editing value cannot be applied to grid properly because dummy cell cannot be focused. 
I think that users can be confused for this operation so that we will discuss about it and fix it for removing ambiguity. 
If it's fixed, I will comment here. Thank you!

**Comment by SanjaySankar** [comment]
@js87zz I tried to do this by using the **on click event** provided by the grid, and then calling the **finishEditing** method inside it. 

Here is the codesandbox link for the example. Please check: 
**https://codesandbox.io/s/crazy-sara-jp93v?file=/src/CompanyTable.jsx**

I tried calling the **startEditing** method using the on click event and it works as intended. So is it a problem with the **finishEditing** method?

**Comment by js87zz** [comment]
@js87zz 
First, I think that you want to start editing after **clicking(not dblclick)** the cell.
If you wan it, you can apply it through the `editingEvent` option.
```js
const grid = new Grid({
  // ...
  data,
  columns,
  editingEvent: 'click' // editing the cell when clicking
})
```
And even If the `finishEditing()` is called in `click` event, the cell value couldn't be changed because the `click` event has been triggered after cancel editing by clicking the dummy cell.

**Comment by SanjaySankar** [comment]
@js87zz 
I'm not looking for editing on a single click. The existing double click functionality is enough, but I want to finish editing and save the new value when I click elsewhere, even if it is a dummy row. Right now when I call `finishEditing()` inside `click` event, it is not working. 
But `startEditing()` works when I do the same thing,even when I click on a dummy row. 

For reference: https://codesandbox.io/s/crazy-sara-jp93v?file=/src/CompanyTable.jsx

**Comment by js87zz** [comment]
@SanjaySankar 
I got it. You just want to finish editing the cell when clicking the dummy cell. 
We will discuss about it and share the result in this issue.

**Comment by HotRedMat** [comment]
@js87zz 
I also have the same problem and can't handle it.
I'd like to finish editing if I'm out of focus on the grid.

**Comment by js87zz** [comment]
@SanjaySankar @HotRedMat 
The bug has been fixed in `v4.14.0` last week(#1088).
Thank you!
